generic/piranha-tree-sitter/src/cleanup_rules/java/rules.toml

```diff
@@ -0,0 +1,484 @@
+# The language specific rules in this file are applied after the API spcific change has been performed.
+
+# Before : 
+#  if (true) { doSomething(); }
+# After :
+#  { doSomething(); } 
+# 
+# Before : 
+#  if (true) { doSomething(); } else { doSomethingElse();}
+# After :
+#  { doSomething(); } 
+#
+[[rules]]
+name = "If statement - true"
+query = """
+(
+    (if_statement
```

Comment:
how is the `else` matched here given this rule seems to handle both cases? with and w/o `else`? For example, don't you want to specify an `alternative` here?